lib/resolve-decl.js

```diff
@@ -71,8 +71,9 @@ function eachMapItemDependencyOfDecl(variablesUsedList, map, decl, cb) {
 
 // Resolve the decl with the computed value
 // Also add in any media queries that change the value as necessary
-function resolveDecl(decl, map, /*optional*/shouldPreserve, /*optional*/logResolveValueResult) {
+function resolveDecl(decl, map, /*optional*/shouldPreserve, /*optional*/logResolveValueResult, /*optional*/preserveAtRulesOrder) {
 	shouldPreserve = shouldPreserve || false;
+	preserveAtRulesOrder = preserveAtRulesOrder || false;
```

Comment:
@awilson82-vp I am in favor of making the default be in order as expected. We can make a major release to play nice for people who have already worked around it

Reply:
@MadLittleMods Could we release this new option as-is now (minor release), and then change default to `true` in the next major release? That would allow us to start using it immediately.
Happy to help out if anything else needs to be done before this can be released.

Reply:
@dampkwab We can do a major release now (no need to wait for anything else). We just need to update the default in this PR.

A minor release is just extra work for no reason.

Reply:
@MadLittleMods I've updated the default now - hope that helps :)
https://github.com/MadLittleMods/postcss-css-variables/pull/100

Reply:
https://github.com/MadLittleMods/postcss-css-variables/pull/100#issuecomment-496321605 is now abandoned

I created https://github.com/MadLittleMods/postcss-css-variables/pull/101 which removes the `playground/` changes so we can build fresh from `master`. It seems like we should have some more tests though. So someone will need to pick that up

If we updated this merge request default to `true`, all of the current tests would need to be updated which would give us nice coverage.

Reply:
@MadLittleMods 
Uff, I really need this fix ASAP =D
What about sticking with harmless preserveAtRulesOrder=false as default until you finish your wished result ^^)

Reply:
@dinodujmovic Sure 🤷‍♀ , now published: https://github.com/MadLittleMods/postcss-css-variables/pull/92#issuecomment-502603507

Reply:
Thanks a lot !